### PR TITLE
Fetch Azure subscription from secret

### PR DESCRIPTION
**Pull request by AverageMarcus**
### Checklist

- [x] Update changelog in CHANGELOG.md.

### CODE REVIEW

**Review by calvix — APPROVED** [review]
lgtm but i have small insight into AZURE

**Review by whites11 — APPROVED** [review]
seems reasonable

**Comment by AverageMarcus** [comment]
Sorry, had to work around Kyverno a little more (no clue how I thought it was working in my initial testing)

I'm going to propose a coalesce function for Kyverno, similar to how we did base64, to make this fallback logic nicer.

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Changed
+
+- Get Azure subscription ID from secret
```
Comment (whites11) [inline CHANGELOG.md:12]:
```suggestion
- Default Azure subscription ID by getting value directly from organization credentials secret.
```

**Review by whites11 — APPROVED** [review]

**Review by fiunchinho — COMMENTED** [review]
If it's not too hard to add, can we add a test for this?

**Review by calvix — APPROVED** [review]

**Pull request merged**